src/stylesheets/components/_accordions.scss

```diff
@@ -90,3 +90,7 @@ $accordion-border: 3px solid $color-gray-lightest;
     margin-bottom: 0;
   }
 }
+
+.usa-accordion-content-collapsed {
```

Comment:
Is there a reason we need the accordion to have it's own class for display none, or should this be put in a general utility?